src/exports/global.js

```diff
@@ -1,34 +1,34 @@
 define( [
 	"../core"
-], function( jQuery, noGlobal ) {
+], function( jQuery, noGlobal, globalObject ) {
```

Comment:
I thought as much, but was confused by a previous integration error. Still not entirely sure how all of this fits together (e.g. why is noGlobal in the "globals" section of the lint JSON file?)